server/src/internalClusterTest/java/org/opensearch/discovery/single/SingleNodeDiscoveryIT.java

```diff
@@ -178,6 +179,7 @@ public Path nodeConfigPath(int nodeOrdinal) {
                 final ClusterState first = internalCluster().getInstance(ClusterService.class).state();
                 assertThat(first.nodes().getSize(), equalTo(1));
                 assertBusy(() -> mockAppender.assertAllExpectationsMatched());
+                internalCluster().stopNodesAndClients();
```

Comment:
Why do we that? This should be covered by the OpenSearchIntegTestCase check, right?

Reply:
Now, not needed since it is out of cluster now.